.github/workflows/ci.yml

```
@@ -166,7 +173,10 @@ jobs:
           scope: '@open-formulieren'
 
       - name: Install dependencies
-        run: npm ci
+        run: |
+          # update npm
+          npm install -g npm@latest
```

```suggestion
          npm install --global npm@latest
```
To keep it consistent with the commands in the other jobs